nautobot_chatops/workers/nautobot.py

```diff
@@ -133,6 +147,39 @@ def get_filtered_connections(device, interface_ct):
     )
 
 
+def analyze_circuit_endpoints(endpoint):
+    """Analyzes a circuit's endpoint and returns info about what object the endpoint connects to."""
+    if type(endpoint) in [Interface, FrontPort, RearPort]:
+        # Put into format: object.device_name
+        info = f"Device: {endpoint.device.name}  Interface: {endpoint.name}"
```

Comment:
Is it desirable to refer to FrontPort and RearPort as Interfaces? Or might it be better to do something like:

```suggestion
        info = f"Device: {endpoint.device.name}  {endpoint.__class__.__name__}: {endpoint.name}"
```

Reply:
made this change.  Thank you!